examples/next-js-netlify/Earthfile

```diff
@@ -0,0 +1,38 @@
+VERSION 0.6
+FROM --platform=linux/amd64 node:latest
+WORKDIR /app
+
+deps:
+    ARG auth_token
+    ARG site_id
+    # Set environment variables required for building and deploying to Netlify
+    ENV NETLIFY_AUTH_TOKEN=$auth_token
+    ENV NETLIFY_SITE_ID=$site_id
```

Comment:
Wondering if these are declared too early in the build. If the values change, too much cache would be invalidated. I would declare these where they are needed only (e.g. in `+build` and `+deploy`), and as late as possible - in order to preserve as much cache as we can.

Reply:
That is fair. I'll update.